sys/auto_init/netif/auto_init_netdev_raw802154.c

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (C) 2015 Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ *
+ */
+
+/**
+ * @ingroup auto_init_ng_netif
+ * @{
+ *
+ * @file
+ * @brief   Auto initialization for ethernet devices
+ *
+ * @author  Kaspar Schleiser <kaspar@schleiser.de>
```

Comment:
I guess you are not Kaspar ;-)

Reply:
No, but I copied it from the great Kaspar.